### PR TITLE
Update rpm and copr-cli pages

### PR DESCRIPTION
**Pull request by galoget**

### CODE REVIEW

File: deployment/rpm/about.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ $ rpmbuild -bs hello.spec
 Then you can run:
 
 ```bash
-$ mock -r epel-7-i386.cfg hello.src.rpm
+$ mock -r /etc/mock/epel-6-i386.cfg ~/rpmbuild/SRPMS/hello-2.10-1.fc27.src.rpm
```
Comment (pvalena) [inline deployment/rpm/about.md]: Why `epel6`? Also, why `i386` and not `x86_64`? (I just wonder if there are some reasons behind it?)
Also I'd say it's really not needed to use full path.

File: deployment/rpm/about.md [hunk 2]
```diff
@@ -23,10 +23,10 @@ Biggest advantages of packaging your application is:
 Run:
 
 ```bash
-$ sudo dnf install fedora-packager rpmdevtools
+$ sudo dnf install fedora-packager rpmdevtools gcc
```
Comment (pvalena) [inline deployment/rpm/about.md]: I wonder why would you need gcc when you're building in mock if I'm not mistaken.

**Comment by pvalena** [comment]
Thank you for your PR!

**Comment by pvalena** [comment]
@galoget, any news on this?

File: deployment/rpm/about.md (hunk 1 above)
Comment (Conan-Kudo) [reply deployment/rpm/about.md]: This should absolutely be for latest fedora and el, not el6...

File: deployment/rpm/about.md (hunk 2 above)
Comment (ferdnyc) [reply deployment/rpm/about.md]: You're not _only_ building in mock. § Quick Start Guide does end with a suggestion to run `rpmbuild -ba hello.spec`, and GNU Hello is a C program. 

Technically you'll need other build tools besides `gcc` to compile GNU Hello, tho: `make`, `autoconf`, etc. I don't _believe_ the three packages listed will necessarily pull all of those in, if they're not already installed.

**Comment by pvalena** [comment]
`This branch has conflicts that must be resolved
`

File: deployment/rpm/about.md (hunk 1 above)
Comment (pvalena) [reply deployment/rpm/about.md]: > for latest fedora and el, not el6...

Actually, as long as this works *on Fedora* it does not matter.

File: deployment/rpm/about.md (hunk 2 above)
Comment (pvalena) [reply deployment/rpm/about.md]: Ok, let's go with that, then.
Thanks!

File: deployment/rpm/about.md (hunk 1 above)
Comment (pvalena) [reply deployment/rpm/about.md]: Ok, let's assume the Contributor has tested this and the values do not matter in the end as it's just an example.

**Comment by pvalena** [comment]
I've resolved those. I hope I did not miss anything.
Thanks!

**Pull request merged**